packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -144,6 +145,8 @@ export default defineComponent({
   setup (props, { emit, attrs, slots }) {
     const input = ref<HTMLInputElement | typeof VaTextarea | undefined>()
 
+    const { isFocused } = useFocus()
```

Comment:
In this case, `isFocused` will always be `false`.
Otherwise, you either need to pass an HTML element to `useFocus` or export `onFocus` and `onBlur` from `useFocus` to manually control `isFocused`

Reply:
And the same in the cases below.